source/includes/api/_before_you_get_started.md

```diff
@@ -1,27 +1,60 @@
 # Before you get started
 
-This section covers a list of topics you should be familiar with before using the API. These concepts span the whole API.
+This section covers a list of topics you should be familiar with before using the API. The concepts described in this section span across multiple API endpoints and are fundamental for a better understanding of how to interact with the RW API.
 
 ## Applications
 
-TODO
+As you might have come across while reading these docs, different applications and websites rely on the RW API as the principal source for their data. While navigating through the catalog of available datasets, you will find some datasets used by the [Resource Watch website](https://resourcewatch.org/), others used by [Global Forest Watch](https://www.globalforestwatch.org/). In many cases, applications even share the same datasets!
 
+To ensure the correct separation of content across the different applications that use the RW API, you will come across a field named `application` in many of the API's resources (such as datasets, layers, widgets, and others). Using this field, the RW API allows users to namespace every single resource, so that it's associated only with the applications that use it.
 
-## Authentication
+### Existing applications
 
-TODO
+Currently, the following applications are using the API as the principal source for their data:
 
+* the [Resource Watch website](https://resourcewatch.org/), where the `application` field takes the value `rw`;
+* the [Global Forest Watch website](https://www.globalforestwatch.org/), where the `application` field takes the value `gfw`;
+* the [Partnership for Resilience and Preparedness website](https://prepdata.org/), where the `application` field takes the value `prep`;
+* the [Forest Atlases websites](https://www.wri.org/our-work/project/forest-atlases) for different countries of the world also rely on the RW API - in the case of these websites, the `application` field takes the value `forest-atlas`;
 
-## Roles
+If you would like to see your application added to the list of applications supported by the RW API, please contact us.
```

Comment:
How?

Reply:
Good question :thinking: we can forward users to [WRI's main contact page](https://www.wri.org/about/contact)... Another option would be to set up some API contact email. @tiagojsag any thoughts on how this kind of communication happened in the past?

Reply:
This was one of the things we had in mind a few months back when we talked about improving the API's homepage to be more informative. Right now AFAIK there's no "contact us" path for anyone to reach out to us about things like this, bugs, questions, etc.IMO this is something we should have.